.travis.yml

```diff
@@ -9,8 +9,7 @@ jobs:
       before_script:
         - ./tasks/wiredeps
```

Comment:
We do not do that on GitHub actions right now, should we?

Reply:
Yes, we want the same behavior we previously had on Travis.

Reply:
O.k. need to add this one then.